cl_dll/hud_crosshairs.cpp

```diff
@@ -108,6 +109,19 @@ int CHudCrosshairs::Draw(float time)
 			gl.line(Vector2D(center.x - offset.x, center.y - offset.y + half_width), Vector2D(center.x - offset.x, center.y + offset.y - half_width));
 			gl.line(Vector2D(center.x - offset.x - half_width, center.y + offset.y), Vector2D(center.x + offset.x + half_width, center.y + offset.y));
 		}
+
+		// Circle
+		if (cl_cross_circle_radius->value && cl_cross_circle_thickness->value > 0.0f) {
```

Comment:
Why did you put this here and not in the circle drawing if below? This block does the exact same thing as the one below except with `cl_cross_outline->value` added to thickness (why?).

Reply:
I wanted to separate the circle's outline like the other outlines are. It just draws a thicker (with outline value added to normal thickness) circle that works as an outline I guess. Would there be a better way of doing this?

Reply:
Oh, I see.